.travis.yml

```diff
@@ -36,21 +36,13 @@ after_success:
   - scripts/deploy.sh netlify
   - scripts/push-image.sh unlock
   - scripts/push-image.sh unlock-integration
-  - scripts/package_locksmith.sh
 after_failure:
   - scripts/docker-compose-log.sh
 notifications:
   slack:
     secure: q4BMSH3FifcaoPwQ2QFocbv8OuNT4JWgeJuE7jxppw86rCJHOyp6SdLpzp4944cWTlEGFdbQsp83OE2fx99i9OncvlJM5Is1IQgIDUfOD2l8mKx+HPlWvuBLaRB5j7vdamzODaHU2NlS+XSU7I0ytxp39x5arYY5hR9yqETvkyjAsSWBjY74Vv+eED+ALtQD9HKoH3xaxgV5rolujR3MEzdMSoNpenivz+BIfKhGQlOHyEN0FOsZtBkCmJTZ9mQbPApu2NKQZRU2cTn7h4Pp4Gpy6mVPWtesUd9OtblsA658qw9Sn/C9hIO1uKgGb4G1PJrKaa/qs2WeMj3VNQ0CkubXYNUS+OfnRb0CT+eTRCndnhFsDdt0zgIBtFeG2FT83OD0sT1SHduY69uETxxstTJw6F8+q/n4JNImBT7klBn2tLrS8kOpk2dqIkpQsvKst67UaYVNa4x2wOcbBxwE+RXP3i3Hs+NpNW/3W9IJOXA2Twx6Q70ORlg3Lnfw28gTDOnh3ZYx48b8y17U4J4KJgEu7aaQCUtoPK/dSwGl0atMQn/FRwEHvIs7zIh/PqyWYRA+y0V3VAe2bAuLf5WTjUijN5w+wfHDGjo6fxZ6dPwBXhhbeFWl4Tz6GEiOGJsebJTtDKWYD/uItNqb3uFiEwEeUEFJCY1zVU5jT8NL3TY=
 deploy:
-  provider: elasticbeanstalk
-  access_key_id:
-    secure: "jwqleo0h2qdUQ9w3+BkLrVKuXHNcY9072XGYFjuxJNkYJxfCBODaanm90hNcNNal6fqV/fkb+ilcvad4DBfpbUdkVTx+YSId8zEL1W99D1G1rJ28vs0nl+RYfbYutQUCvAWWD0wpTLDmcPNOFUBJc/RIiEeeqAywOueHAV1K4srYW3IjJWYskb0EdW+hANmLAnNkKboiPWj7FH9HoizH+Owt3apBdAkJ+4efKyyRCIZmzGPF3ymLezHhiWWu57WQtM4yo+1ipvqCQkzaC+rmVf+1YZUXf5HXeuhmyp+A4vn8x6Oy3V+PVwazh1ro4ylxpRXKMd/ia59iQ3QNcUsPlfE3Ew9g98hKKW9/QMwRCUxenp9ThmDwXvzoy29oMI+7xag03IiOhjR8gaKYADlt8vuaCwTuKruuKNgnfeBKhJH+Batgji508SbvNZ+M18W/4W3gn3JLbBiabpcTO/zpTbKwfUnAfBpsPNWwoQ9GlwSLF9FYapaS61ycYvKXCNTX/fsvB59aH9cD/H9jO65L7E9Y/fZbR/K+HoFEDKj43ZKUg/H68hwDLMKgx+legv8v3J6T2/XLPOdAw2E4pdRlIMkWSotithO5LPSsV4JXHlw148G41Cs+NRvcfj2tbqTZgEy+7ZsgFT3FV2nhfK6lFlYSr6vJUX9pOHA01d7Rjc4="
-  secret_access_key:
-    secure: "TP5OBYbSCSjONSFodWijHKcrkxlVPPE5JEkwzXiO5wDgfEo0s8gGA/hci49tm2vURW25JaFDUtuYkN8YHxCrNCjoLrOgPGKT9i3xJJmKEa3gJu4IwShk1hhWebe8XKSq7EoB03xbjoMab3zUIzI3dTsNTz2thV2cuZFO6hEQjIZoYpMd18P2O3HNUgUV1ubfE9fHFcDAEP1HthTEMBBWsUvKDe6HnqK69ef4KKSl+CjGdC88XZJqhtHENr2qQF8QYYhkVWJcNsFZAfFHsW9WKxxyZVDZfQNraYWGjpvH0brJ4uv6mvTvHNPfngUittSdMivGthpvJzx7qBJ7tfX+z2iBa4MlcrJZktM6Id/4exOr4p8mZKfCEJ0V0Lbn3dZm60jopFx3Krg996r8ry03gUfYW05OxTXH6grSjSAwfCaswHyOqKRZXOwCcZ8/zMYc5a8aVo4bJk2z2NKZBFBmayx+mPe0lckKlJB5ElfpGTWG5NLHyKtSYXeVioY0ebleHWJTAQ29fbPMC/eA6bPirEzusUD5OMXE5GBMywJ+f8wQIxftyku3lVstV+tMvFJTxmpwASLh+L8u19GGTgo9XRksm8xC+Ai2+2VBh9sbtf1JhJokS/feYW0ntVL4dJDfzBVW42SARjfN4GDQ5iEZ3Fy0ps84CPHij33lEpcZOdI="
-  region: "us-east-1"
-  app: "locksmith"
-  env: "Locksmith-env-1"
-  skip_cleanup: true
-  zip_file: "./builds/locksmith.zip"
-  bucket_name: "unlock-locksmith"
+  provider: script
```

Comment:
Ha! I think we should not use the travis logic here but add `scripts/deploy-elasticbeanstalk.sh` inside the `after_success:` block above.

Reply:
Totally make sense, this will be removed before I take the PR out of draft mode